packages/contracts-bedrock/src/L2/L1BlockHolocene.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { L1BlockInterop, ConfigType } from "src/L2/L1BlockInterop.sol";
+import { StaticConfig } from "src/libraries/StaticConfig.sol";
+import "src/libraries/L1BlockErrors.sol";
+
+/// @custom:proxied
+/// @custom:predeploy 0x4200000000000000000000000000000000000015
+/// @title L1BlockHolocene
+/// @notice Interop extenstions of L1Block.
+contract L1BlockHolocene is L1BlockInterop {
```

Comment:
We don't want to target `L1BlockInterop` here since this will go to production before interop. I think naming this `L1BlockHolocene` for now is safer, even though I think this could technically go right into the `L1Block` contract itself. By not putting this logic into `L1Block` now, it reduces risk that somebody deploys with it before its ready end to end, we will need to pull it into the regular `L1Block` contract in the future.

Reply:
Mark, should we also be modifying SystemConfig.sol directly rather than introducing a SystemConfigHolocene.sol?

Reply:
Good catch, fixed in the latest commit.